pkg/ffuf/job.go

```diff
@@ -499,7 +505,7 @@ func (j *Job) handleGreedyRecursionJob(resp Response) {
 // not been reached
 func (j *Job) handleDefaultRecursionJob(resp Response) {
 	recUrl := resp.Request.Url + "/" + "FUZZ"
-	if (resp.Request.Url + "/") != resp.GetRedirectLocation(true) {
+	if (resp.Request.Url+"/") != resp.GetRedirectLocation(true) && resp.StatusCode >= 300 && resp.StatusCode < 400 {
```

Comment:
I think we should honor the values in the new cli parameter for recursion-status here instead of hardcoded values between 300 and 400.

Reply:
nice catch,  i pushed the suggested change